comptests/s3/test_rest_api_rgw.py

```diff
@@ -180,34 +251,37 @@ def test_user_create_36636(self):
         user_name2 = f"{self.user_name_prefix}{str(time.perf_counter_ns()).replace('.', '_')}"
         user_params = {
             'display-name': user_name,
-            'uid' : user_name
-         }
+            'uid': user_name
+        }
         user_params2 = {
             'display-name': user_name,
-            'uid' : user_name2
-         }
-        self.log.info("Step 1: Creating a new IAM user with name %s", str(user_name))
+            'uid': user_name2
+        }
+        self.log.info(
+            "Step 1: Creating a new IAM user with name %s", str(user_name))
         loop = asyncio.get_event_loop()
         status, user_info = loop.run_until_complete(self.obj.create_user(user_params))
-        assert status == HTTPStatus.OK , "Not able to create user. Test Failed"
-        self.log.info("Created user details: %s",user_info)
-        self.log.info("Step 2: Verifying that new IAM user is created successfully")
+        assert status == HTTPStatus.OK, "Not able to create user. Test Failed"
+        self.log.info("Created user details: %s", user_info)
+        self.log.info(
+            "Step 2: Verifying that new IAM user is created successfully")
         status, user_info = loop.run_until_complete(self.obj.get_user_info(user_params))
-        assert status == HTTPStatus.OK , "Not able to Get user Info. Test Failed"
-        self.log.info("Get user info output: %s",user_info)
+        assert status == HTTPStatus.OK, "Not able to Get user Info. Test Failed"
+        self.log.info("Get user info output: %s", user_info)
         self.created_users.append(user_params)
-        self.log.info("Step 3: Creating another IAM user with same display name %s", str(user_name))
+        self.log.info(
+            "Step 3: Creating another IAM user with same display name %s", str(user_name))
         loop = asyncio.get_event_loop()
         status, user_info = loop.run_until_complete(self.obj.create_user(user_params2))
-        assert status == HTTPStatus.OK , "Not able to create user. Test Failed"
-        self.log.info("Created user details: %s",user_info)
+        assert status == HTTPStatus.OK, "Not able to create user. Test Failed"
+        self.log.info("Created user details: %s", user_info)
         self.log.info(
             "Step 4: Verifying that new IAM user is created successfully with same display name")
         status, user_info = loop.run_until_complete(self.obj.get_user_info(user_params2))
-        assert status == HTTPStatus.OK , "Not able to Get user Info. Test Failed"
-        self.log.info("Get user info output: %s",user_info)
+        assert status == HTTPStatus.OK, "Not able to Get user Info. Test Failed"
```

Comment:
Instead of just relying on the created  user response code , can you verify with get to ensure that resource was actually created. This is adding additional verification to qualify them as public API system tests as well.